### PR TITLE
doc: fix dump cache command

### PR DESCRIPTION
**Pull request by joscollin**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephfs/troubleshooting.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ them. Start by looking to see if either side has stuck operations
 
 We can get hints about what's going on by dumping the MDS cache ::
 
-  ceph daemon mds.<name> dump cache /tmp/dump.txt
+  ceph daemon mds.<name> dump cache > /tmp/dump.txt
```
Comment (batrick) [inline doc/cephfs/troubleshooting.rst:15]: This isn't right? See the note below this command:

**Review by batrick — CHANGES_REQUESTED** [review]

File: doc/cephfs/troubleshooting.rst (hunk 1 above)
Comment (joscollin) [reply doc/cephfs/troubleshooting.rst:15]: I understand that. But when I try with vstart, this command doesn't output the `dump cache` json. 
It outputs the below instead, which is the reason I created this patch. Is this expected?

```
jcollin@smithi173:~/ceph/build$ cat /tmp/dump.txt 
[inode 0x3 [...2,head] #3/ auth v1 snaprealm=0x5580154aea00 f() n(v0 rc2021-02-24T13:46:26.659471+0000 1=0+1) (iversion lock) 0x558015569080]
[inode 0x609 [...2,head] ~mds0/stray9/ auth v1 f() n(v0 rc2021-02-24T13:46:26.658489+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015568b00]
 [dir 0x609 ~mds0/stray9/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x558015574000]
[inode 0x606 [...2,head] ~mds0/stray6/ auth v1 f() n(v0 rc2021-02-24T13:46:26.657265+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015499700]
 [dir 0x606 ~mds0/stray6/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bd180]
[inode 0x605 [...2,head] ~mds0/stray5/ auth v1 f() n(v0 rc2021-02-24T13:46:26.656847+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015499180]
 [dir 0x605 ~mds0/stray5/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bcd00]
[inode 0x603 [...2,head] ~mds0/stray3/ auth v1 f() n(v0 rc2021-02-24T13:46:26.656047+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015498680]
 [dir 0x603 ~mds0/stray3/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bc400]
[inode 0x602 [...2,head] ~mds0/stray2/ auth v1 f() n(v0 rc2021-02-24T13:46:26.655651+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015498100]
 [dir 0x602 ~mds0/stray2/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bbf80]
[inode 0x608 [...2,head] ~mds0/stray8/ auth v1 f() n(v0 rc2021-02-24T13:46:26.658084+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015568580]
 [dir 0x608 ~mds0/stray8/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bda80]
[inode 0x604 [...2,head] ~mds0/stray4/ auth v1 f() n(v0 rc2021-02-24T13:46:26.656439+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015498c00]
 [dir 0x604 ~mds0/stray4/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bc880]
[inode 0x601 [...2,head] ~mds0/stray1/ auth v1 f() n(v0 rc2021-02-24T13:46:26.655228+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015497b80]
 [dir 0x601 ~mds0/stray1/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bbb00]
[inode 0x607 [...2,head] ~mds0/stray7/ auth v1 f() n(v0 rc2021-02-24T13:46:26.657684+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015568000]
 [dir 0x607 ~mds0/stray7/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bd600]
[inode 0x600 [...2,head] ~mds0/stray0/ auth v1 f() n(v0 rc2021-02-24T13:46:26.654726+0000 1=0+1) (iversion lock) | dirfrag=1 stickydirs=1 stray=1 dirtyparent=0 authpin=0 0x558015497600]
 [dir 0x600 ~mds0/stray0/ [2,head] auth v=1 cv=1/1 state=1073750017|complete|sticky f() n() hs=0+0,ss=0+0 | sticky=1 dirty=0 authpin=0 0x5580154bb680]
[inode 0x100 [...2,head] ~mds0/ auth v1 snaprealm=0x5580154ae800 f(v0 10=0+10) n(v0 rc2021-02-24T13:46:26.658489+0000 11=0+11) (iversion lock) | dirfrag=1 0x558015497080]
 [dir 0x100 ~mds0/ [2,head] auth v=1 cv=1/1 dir_auth=0 state=1073741825|complete f(v0 10=0+10) n(v0 rc2021-02-24T13:46:26.658489+0000 10=0+10) hs=10+0,ss=0+0 | child=1 subtree=1 dirty=0 authpin=0 0x5580154bb200]
  [dentry #0x100/stray6 [2,head] auth (dversion lock) v=0 ino=0x606 state=1073741824 | inodepin=1 dirty=0 0x5580147eaf80]
  [dentry #0x100/stray7 [2,head] auth (dversion lock) v=0 ino=0x607 state=1073741824 | inodepin=1 dirty=0 0x5580147eb480]
  [dentry #0x100/stray0 [2,head] auth (dversion lock) v=0 ino=0x600 state=1073741824 | inodepin=1 dirty=0 0x5580147e9180]
  [dentry #0x100/stray9 [2,head] auth (dversion lock) v=0 ino=0x609 state=1073741824 | inodepin=1 dirty=0 0x558015578000]
  [dentry #0x100/stray5 [2,head] auth (dversion lock) v=0 ino=0x605 state=1073741824 | inodepin=1 dirty=0 0x5580147eaa80]
  [dentry #0x100/stray2 [2,head] auth (dversion lock) v=0 ino=0x602 state=1073741824 | inodepin=1 dirty=0 0x5580147e9b80]
  [dentry #0x100/stray1 [2,head] auth (dversion lock) v=0 ino=0x601 state=1073741824 | inodepin=1 dirty=0 0x5580147e9680]
  [dentry #0x100/stray8 [2,head] auth (dversion lock) v=0 ino=0x608 state=1073741824 | inodepin=1 dirty=0 0x5580147eb980]
  [dentry #0x100/stray3 [2,head] auth (dversion lock) v=0 ino=0x603 state=1073741824 | inodepin=1 dirty=0 0x5580147ea080]
  [dentry #0x100/stray4 [2,head] auth (dversion lock) v=0 ino=0x604 state=1073741824 | inodepin=1 dirty=0 0x5580147ea580]
[inode 0x1 [...2,head] / auth v1 snaprealm=0x5580154ae600 f() n(v0 rc2021-02-24T13:46:26.654091+0000 1=0+1) (iversion lock) | dirfrag=1 0x558015496b00]
 [dir 0x1 / [2,head] auth v=1 cv=1/1 REP dir_auth=0 state=1073741825|complete f() n() hs=0+0,ss=0+0 | subtree=1 dirty=0 authpin=0 0x5580154bad80]
```

Comment (joscollin) [reply doc/cephfs/troubleshooting.rst:15]: I might be missing something here, so trying to clarify.

Comment (batrick) [reply doc/cephfs/troubleshooting.rst:15]: I'm not sure what you're trying to fix.

**Pull request closed without merging**